ephios/templates/registration/login.html

```diff
@@ -10,8 +10,9 @@
     {% trans "Login" as login_message %}
     <h2>{{ login_message }}</h2>
     <form method="post">
-      {% csrf_token %}
-      {% bootstrap_form form %}
-      {% bootstrap_button login_message "submit" %}
+        {% csrf_token %}
+        {% bootstrap_form form %}
+        {% bootstrap_button login_message "submit" %}
+        <a class="float-right" href="{% url "password_reset" %}">{% translate "Forgot your password?" %}</a>
```

Comment:
Can be outside the form, right?

Reply:
no, it cannot because it looks like shit then